project.clj

```diff
@@ -13,6 +13,7 @@
             [lein-cljfmt "0.8.0" :exclusions [org.clojure/clojure]]]
   :dependencies [[org.clojure/clojure "1.10.3"]
                  [io.github.protojure/core "2.0.1"]
+                 [io.github.protojure/grpc-client "2.0.1"]
```

Comment:
I think the latest is v2.0.9 so might as well bump these up, while you are adding a new one.  I know you were just matching the core 2.0.1

Reply:
Done & Done!